Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rs-consul"
-version = "0.1.1"
+version = "1.0.0"
```

Comment:
Can we go to 0.2.0? I think we want to reserve 1.0.0 for when we're feature-complete.